extensions/api/data-management/transferprocess/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/service/TransferProcessServiceImpl.java

```diff
@@ -72,6 +75,18 @@ public TransferProcessServiceImpl(TransferProcessStore transferProcessStore, Tra
         return apply(transferProcessId, this::deprovisionImpl);
     }
 
+    @Override
+    public @NotNull ServiceResult<String> initiateTransfer(DataRequest request) {
+        return transactionContext.execute(() -> {
+            TransferInitiateResult transferInitiateResult = manager.initiateConsumerRequest(request);
+            return Optional.ofNullable(transferInitiateResult)
+                    .filter(AbstractResult::succeeded)
+                    .map(AbstractResult::getContent)
+                    .map(ServiceResult::success)
+                    .orElse(ServiceResult.conflict("Request couldn't be initialised."));
```

Comment:
Not sure if use of `conflict` is correct here, As it is used at other places when we have something already exist. But I also see that `ServiceResult` doesn't offer anything else here.
So maybe we can go with this one and see what EDC upstream devs suggest here.

Reply:
agree, the other option is notFound. And at first I didn't wrap processId in ServiceResult but it was suggested by Andrea to be consistent with other methods in the controller.